libs/index/odc/index/stac.py

```diff
@@ -29,8 +33,23 @@ def _stac_product_lookup(item):
                 properties['sentinel:latitude_band'],
                 properties['sentinel:grid_square']
             )
+            product_id = str(odc_uuid("sentinel-2_stac_process", "1.0.0", [product_label]))
+    elif properties.get('platform') in LANDSAT_PLATFORMS:
+        product_id = item['id']
+        product_label = _product_label(item)
+        product_name = properties.get('odc:product')
+        region_code = properties.get('odc:region_code')
```

Comment:
Needs and `else:` with reasonable something in it.

Reply:
Added `else:` statement